database/models/reports.py

```diff
@@ -322,3 +322,28 @@ class TestResultReportTotals(CodecovBaseModel, MixinBaseClass):
     skipped = Column(types.Integer)
     failed = Column(types.Integer)
     error = Column(types.String(100), nullable=True)
+
+
+class ReducedError(CodecovBaseModel, MixinBaseClass):
+    __tablename__ = "reports_reducederror"
+    message = Column(types.Text)
+
+
+class Flake(CodecovBaseModel, MixinBaseClass):
```

Comment:
Is the idea to then replace these w/ their django counterparts after you run the risky migration?

Reply:
no, these will remain, these changes just don't depend on the risky migrations to run

Reply:
Mmm is there a reason we don't want to use the django models instead?

Reply:
it's possible the `Flake` model is not needed, but the `ReducedError` model is needed because we will be writing the reduced_error_id in the test results processor in future changes

Reply:
After talking, we'll defer the use of django models as it poses some difficulties with testing atm